conda_build/metadata.py

```diff
@@ -1665,7 +1666,6 @@ def build_id(self):
             raise RuntimeError(
                 f"Couldn't extract raw recipe text for {self.name()} output"
             )
-        raw_recipe_text = self.extract_package_and_build_text()
```

Comment:
I wonder how much of a speedup removing this gives us ([since both of us did this](https://github.com/conda/conda-build/pull/5237#discussion_r1526780334))